pkg/controller/v1beta1/inferenceservice/rawkube_controller_test.go

```diff
@@ -9143,7 +9143,7 @@ var _ = Describe("v1beta1 inference service controller", func() {
 								FSGroupChangePolicy: nil,
 								SeccompProfile:      nil,
 							},
-							AutomountServiceAccountToken: proto.Bool(false),
+							AutomountServiceAccountToken: proto.Bool(true),
```

Comment:
Can you add a comment saying this should be true for ODH only? It helps to have comments where things are intentionally different for the sync